Clover/app/src/main/java/org/floens/chan/ui/controller/AppearanceSettingsController.java

```diff
@@ -81,6 +81,10 @@ private void populatePreferences() {
                     R.string.setting_enable_reply_fab,
                     R.string.setting_enable_reply_fab_description)));
 
+            requiresUiRefresh.add(layout.add(new BooleanSettingView(this,
+                    ChanSettings.accessibleInfo,
+                    "Enable accessible post info",
+                    "Place post info in a more accessible location")));
```

Comment:
you should put these raw strings in [`strings.xml`](https://github.com/Floens/Clover/blob/3c7f0ddfff53d179703c78248c864ceb68acd67f/Clover/app/src/main/res/values/strings.xml)